app/controllers/queue_items_controller.rb

```diff
@@ -19,6 +31,24 @@ def destroy
 
   private
 
+  def update_reviews(queue_ids_with_ratings)
+    queue_ids_with_ratings.each do |id, rating|
+      queue_item = QueueItem.find(id)
+      review = Review.find_by(user: current_user, video: queue_item.video)
+      if review
+        review.update(rating: rating.to_i)
+      else
+        review = Review.new(user: current_user, video: queue_item.video, rating: rating)
+        review.save(validate: false)
+      end
+    end
+  end
+
+  def queue_items_have_same_positions?(items)
+    positions = items.flatten.select.each_with_index { |_, i| i.odd? }
+    positions.length != positions.uniq.length ? true : false
```

Comment:
`? true : false` is not needed here.

HOMEWORK: analyze why `? true : false` is a bad choice.